Tests/AppTests/WebpageSnapshotTests.swift

```diff
@@ -13,6 +13,15 @@ class WebpageSnapshotTests: XCTestCase {
     override func setUpWithError() throws {
         Current.date = { Date(timeIntervalSince1970: 0) }
     }
+    
+    override func tearDownWithError() throws {
+        // Remove any HTML files in bundle
+        let fileManager = Foundation.FileManager.default
+        let indexURL = baseURL().appendingPathComponent("index.html")
+        if fileManager.fileExists(atPath: indexURL.path) {
+            try fileManager.removeItem(at: indexURL)
+        }
```

Comment:
Might be worth sticking these in `/tmp` to avoid stuff being left behind in case of test crashes. I should have something like that in another project.

Reply:
This may be possible, but it would also mean that we have to copy the entire `Public` directory to the same tmp directory at the start of the test which could be intensive - especially if we use a different directory for every test.

I got around that duplication, in essence, by copying the files as test resources and using the bundle. I could move this script in with the snapshot extension to guarantee it's done before a new snapshot is created which guarantees success no matter if a previous run crashed?

Of course if you ever clear your derived data it's all cleared up too with the bundle being stored there.

Reply:
I’m going to look into this a little more to see if we can copy the public directory just once.  Will update you on whether this works in the morning 👍